spec/log_weasel/sidekiq_spec.rb

```diff
@@ -23,22 +23,28 @@ def perform
 
   describe "initialize!" do
     after :each do
-      Sidekiq.client_middleware.clear
-      Sidekiq.server_middleware.clear
+      Sidekiq.configure_client do |sidekiq|
+        sidekiq.client_middleware.clear
+        sidekiq.server_middleware.clear
+      end
     end
 
     it "sets the client middleware" do
       described_class.initialize!
 
-      expect(Sidekiq.client_middleware.map(&:klass)).to contain_exactly(described_class::ClientMiddleware)
+      Sidekiq.configure_client do |sidekiq|
+        expect(sidekiq.client_middleware.map(&:klass)).to contain_exactly(described_class::ClientMiddleware)
+      end
```

Comment:
I'm not as familiar with Sidekiq internals, so am actually hoping there's a better way to do this 🤞

Reply:
For now, I think it's fine.